src/core/model_config.proto

```diff
@@ -624,6 +642,19 @@ message ModelOptimizationPolicy
       //@@         input without batching dimension.
       //@@
       map<string, Shape> input = 2;
+
+      //@@      .. cpp:var:: LowerBound graph_lower_bound
```

Comment:
Looks good. Just add optional, as in "Specify the lower bound of the CUDA graph. Optional. If specfied..."